Packs/SplunkPy/Layouts/layoutscontainer-Splunk_Notable_Generic.json

```diff
@@ -0,0 +1,649 @@
+                        "y": 0
+                    },
+                    {
+                        "displayType": "CARD",
+                        "h": 3,
+                        "hideName": false,
+                        "i": "caseinfoid-dwdtwscjnm-field-changed-dwdtwscjnm-caseinfoid-dwdtwscjnm-field-changed-caseinfoid-dwdtwscjnm-field-changed-dwdtwscjnm-caseinfoid-547bd270-abea-11eb-8990-dba25d1c608d",
+                        "items": [
+                            {
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "tags",
+                                "height": 53,
+                                "id": "74ab6a60-abea-11eb-8990-dba25d1c608d",
+                                "index": 0,
+                                "listId": "caseinfoid-547bd270-abea-11eb-8990-dba25d1c608d",
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "srcs",
+                                "height": 53,
+                                "id": "f3689ef0-b0f8-11eb-aa15-bb26b139b89e",
+                                "index": 2,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 4,
+                                "fieldId": "usernames",
+                                "height": 53,
+                                "id": "eeef4bd0-b0f8-11eb-aa15-bb26b139b89e",
+                                "index": 0,
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            },
+                            {
+                                "endCol": 4,
+                                "fieldId": "dsts",
+                                "height": 22,
+                                "id": "f62d1620-b0f8-11eb-aa15-bb26b139b89e",
+                                "index": 1,
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            }
+                        ],
+                        "maxH": null,
+                        "maxW": 3,
+                        "minH": 1,
+                        "minW": 2,
+                        "moved": false,
+                        "name": "Common CIM Fields",
+                        "static": false,
+                        "w": 2,
+                        "x": 1,
+                        "y": 0
+                    },
+                    {
+                        "displayType": "CARD",
+                        "description": "‘True’ indicates enrichment was successful, even if there were no results found.",
```

Comment:
‘True’ indicates enrichment was successful, even if there were no results found. 
See Assets, Identities and Drilldown tabs for details.